cloud/pkg/cloudstream/containermetrics_connection.go

```diff
@@ -74,8 +74,8 @@ func (ms *ContainerMetricsConnection) SendConnection() (stream.EdgedConnection,
 		URL:    *ms.r.Request.URL,
 		Header: ms.r.Request.Header,
 	}
-	connector.URL.Scheme = "http"
-	connector.URL.Host = net.JoinHostPort("127.0.0.1", fmt.Sprintf("%v", constants.ServerPort))
+	connector.URL.Scheme = httpScheme
+	connector.URL.Host = net.JoinHostPort(defaultServerHost, fmt.Sprintf("%v", constants.ServerPort))
```

Comment:
please rebase it.
I have change it in https://github.com/kubeedge/kubeedge/pull/2125/files